client/opentracing/src/main/java/org/hawkular/apm/client/opentracing/DeploymentMetaData.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.apm.client.opentracing;
+
+import org.hawkular.apm.api.utils.PropertyUtil;
+
+/**
+ * @author Juraci Paixão Kröhling
+ */
+public class DeploymentMetaData {
+    private static String ENV_SERVICE_NAME = getServiceNameFromEnv();
+    private static String ENV_BUILD_STAMP = getBuildStampFromEnv();
+    private static DeploymentMetaData INSTANCE = new DeploymentMetaData(ENV_SERVICE_NAME, ENV_BUILD_STAMP);
+
+    private String name;
```

Comment:
Could you change this to 'service', or 'serviceName' as that is how it is known in other places - to avoid any confusion.